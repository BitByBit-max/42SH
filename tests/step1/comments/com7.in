echo idk what to write anymore
if true; then echo hi; #kill me
fi
