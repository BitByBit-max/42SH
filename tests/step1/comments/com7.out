idk what to write anymore
hi
